src/vivarium/framework/lookup/table.py

```diff
@@ -131,23 +144,22 @@ def __init__(
         param_cols_with_edges = []
         for p in parameter_columns:
             param_cols_with_edges += [(p, f"{p}_start", f"{p}_end")]
-        # We manually remove 'year' from the view columns since it is not an attribute
-        # but rather we compute it dynamically
-        view_columns = sorted((set(key_columns) | set(parameter_columns)) - {"year"})
 
         self.parameter_columns_with_edges = param_cols_with_edges
 
+        value_columns_list = (
+            [value_columns] if isinstance(value_columns, str) else list(value_columns)
+        )
         required_cols = (
             set(self.key_columns)
             | set([col for p in self.parameter_columns_with_edges for col in p])
-            | set(self.value_columns)
+            | set(value_columns_list)
         )
-        extra_columns = self.data.columns.difference(list(required_cols))
-
-        if not self.value_columns:
```

Comment:
I'm not seeing how this bit is now being handled.

Reply:
It should never be empty anymore.